pipeline/frontend/yaml/types/container.go

```diff
@@ -38,26 +38,21 @@ type (
 		Privileged bool `yaml:"privileged,omitempty"`
 
 		// Undocumented
-		CapAdd        []string            `yaml:"cap_add,omitempty"`
-		CapDrop       []string            `yaml:"cap_drop,omitempty"`
-		CPUQuota      base.StringOrInt    `yaml:"cpu_quota,omitempty"`
-		CPUSet        string              `yaml:"cpuset,omitempty"`
-		CPUShares     base.StringOrInt    `yaml:"cpu_shares,omitempty"`
-		Devices       []string            `yaml:"devices,omitempty"`
-		DNSSearch     base.StringOrSlice  `yaml:"dns_search,omitempty"`
-		DNS           base.StringOrSlice  `yaml:"dns,omitempty"`
-		ExtraHosts    []string            `yaml:"extra_hosts,omitempty"`
-		IpcMode       string              `yaml:"ipc_mode,omitempty"`
-		Isolation     string              `yaml:"isolation,omitempty"`
-		MemLimit      base.MemStringOrInt `yaml:"mem_limit,omitempty"`
-		MemSwapLimit  base.MemStringOrInt `yaml:"memswap_limit,omitempty"`
-		MemSwappiness base.MemStringOrInt `yaml:"mem_swappiness,omitempty"`
-		NetworkMode   string              `yaml:"network_mode,omitempty"`
-		Networks      Networks            `yaml:"networks,omitempty"`
-		ShmSize       base.MemStringOrInt `yaml:"shm_size,omitempty"`
-		Sysctls       base.SliceOrMap     `yaml:"sysctls,omitempty"`
-		Tmpfs         []string            `yaml:"tmpfs,omitempty"`
-		Ulimits       Ulimits             `yaml:"ulimits,omitempty"`
+		CPUQuota     base.StringOrInt    `yaml:"cpu_quota,omitempty"`
+		CPUSet       string              `yaml:"cpuset,omitempty"`
+		CPUShares    base.StringOrInt    `yaml:"cpu_shares,omitempty"`
+		Devices      []string            `yaml:"devices,omitempty"`
+		DNSSearch    base.StringOrSlice  `yaml:"dns_search,omitempty"`
+		DNS          base.StringOrSlice  `yaml:"dns,omitempty"`
+		ExtraHosts   []string            `yaml:"extra_hosts,omitempty"`
+		IpcMode      string              `yaml:"ipc_mode,omitempty"`
+		MemLimit     base.MemStringOrInt `yaml:"mem_limit,omitempty"`
+		MemSwapLimit base.MemStringOrInt `yaml:"memswap_limit,omitempty"`
+		NetworkMode  string              `yaml:"network_mode,omitempty"`
+		Networks     Networks            `yaml:"networks,omitempty"`
+		ShmSize      base.MemStringOrInt `yaml:"shm_size,omitempty"`
+		Sysctls      base.SliceOrMap     `yaml:"sysctls,omitempty"`
+		Tmpfs        []string            `yaml:"tmpfs,omitempty"`
```

Comment:
Crazy so all left options are actually supported?

Reply:
I did not follow them ... some are still dead code but need more refactoring - so for the next pull